From 972f199faf3ee091588565291d20cb609c9dcd81 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Tue, 1 Sep 2015 09:58:02 +0800
Subject: [PATCH 1293/1594] MLK-11465-2: ARM: dts: imx7d/imx6ul: update SW1A
 for PFUZE3000

commit 58c3b2b0a8a2e6720cce5083d19bb943b2764980 from
git://git.freescale.com/imx/linux-2.6-imx.git

Update max voltage of SW1A to 3.3V on PF3000, see below datasheet:
http://cache.freescale.com/files/analog/doc/data_sheet/PF3000.pdf?fsrch=1&sr=1&pageNum=1

Signed-off-by: Robin Gong <b38343@freescale.com>
---
 arch/arm/boot/dts/imx6ul-9x9-evk.dts          |    8 ++++++++
 arch/arm/boot/dts/imx7d-12x12-ddr3-arm2.dts   |    2 +-
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |    2 +-
 arch/arm/boot/dts/imx7d-19x19-ddr3-arm2.dts   |    2 +-
 arch/arm/boot/dts/imx7d-19x19-lpddr3-arm2.dts |    2 +-
 arch/arm/boot/dts/imx7d-sdb.dts               |    2 +-
 6 files changed, 13 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-9x9-evk.dts b/arch/arm/boot/dts/imx6ul-9x9-evk.dts
index 0e0fcbe..97ebeed 100644
--- a/arch/arm/boot/dts/imx6ul-9x9-evk.dts
+++ b/arch/arm/boot/dts/imx6ul-9x9-evk.dts
@@ -237,6 +237,14 @@
 		reg = <0x08>;
 
 		regulators {
+			sw1a_reg: sw1a {
+					regulator-min-microvolt = <700000>;
+					regulator-max-microvolt = <3300000>;
+					regulator-boot-on;
+					regulator-always-on;
+					regulator-ramp-delay = <6250>;
+			};
+
 			/* use sw1c_reg to align with pfuze100/pfuze200 */
 			sw1c_reg: sw1b {
 				regulator-min-microvolt = <700000>;
diff --git a/arch/arm/boot/dts/imx7d-12x12-ddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-ddr3-arm2.dts
index 25266a1..96204b9 100644
--- a/arch/arm/boot/dts/imx7d-12x12-ddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-ddr3-arm2.dts
@@ -487,7 +487,7 @@
 		regulators {
 			sw1a_reg: sw1a {
 				regulator-min-microvolt = <700000>;
-				regulator-max-microvolt = <1475000>;
+				regulator-max-microvolt = <3300000>;
 				regulator-boot-on;
 				regulator-always-on;
 				regulator-ramp-delay = <6250>;
diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index c50fbd1..ae4faca 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -884,7 +884,7 @@
 		regulators {
 			sw1a_reg: sw1a {
 				regulator-min-microvolt = <700000>;
-				regulator-max-microvolt = <1475000>;
+				regulator-max-microvolt = <3300000>;
 				regulator-boot-on;
 				regulator-always-on;
 				regulator-ramp-delay = <6250>;
diff --git a/arch/arm/boot/dts/imx7d-19x19-ddr3-arm2.dts b/arch/arm/boot/dts/imx7d-19x19-ddr3-arm2.dts
index 4790a3f..3186791 100644
--- a/arch/arm/boot/dts/imx7d-19x19-ddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-19x19-ddr3-arm2.dts
@@ -652,7 +652,7 @@
 		regulators {
 			sw1a_reg: sw1a {
 				regulator-min-microvolt = <700000>;
-				regulator-max-microvolt = <1475000>;
+				regulator-max-microvolt = <3300000>;
 				regulator-boot-on;
 				regulator-always-on;
 				regulator-ramp-delay = <6250>;
diff --git a/arch/arm/boot/dts/imx7d-19x19-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-19x19-lpddr3-arm2.dts
index 46eac56..5e65839 100644
--- a/arch/arm/boot/dts/imx7d-19x19-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-19x19-lpddr3-arm2.dts
@@ -382,7 +382,7 @@
 		regulators {
 			sw1a_reg: sw1a {
 				regulator-min-microvolt = <700000>;
-				regulator-max-microvolt = <1475000>;
+				regulator-max-microvolt = <3300000>;
 				regulator-boot-on;
 				regulator-always-on;
 				regulator-ramp-delay = <6250>;
diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 1d8fd74..75be101 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -256,7 +256,7 @@
 		regulators {
 			sw1a_reg: sw1a {
 				regulator-min-microvolt = <700000>;
-				regulator-max-microvolt = <1475000>;
+				regulator-max-microvolt = <3300000>;
 				regulator-boot-on;
 				regulator-always-on;
 				regulator-ramp-delay = <6250>;
-- 
1.7.5.4

